lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+  MENU = {
+    burger: 7,
+    hamburger: 7,
+    chips: 2
+  }
+  attr_accessor :dishes
+
+  def initialize(dishes= MENU)
+    @dishes = dishes
+  end
+
+  def see_menu
+    @dishes.map { |dish, price| "#{dish} = £#{price}"}.join(', ')
```

Comment:
@dishes.map { |dish, price| "#{dish} = £#{price}" }.join(', '). Missing space between " and }